lib/license_finder/package_manager.rb

```diff
@@ -103,10 +103,10 @@ def current_packages_with_relations
     end
 
     def self.command_exists?(command)
-      if LicenseFinder::Platform.windows?
-        _stdout, _stderr, status = Cmd.run("where #{command} 2>NUL")
+      _stdout, _stderr, status = if LicenseFinder::Platform.windows?
```

Comment:
Hey, just nitpicking but for readability and uniform indentation, can you move if statement to the next line.